src/app/(app)/missing-shoes/components/MissingShoes.tsx

```diff
@@ -0,0 +1,163 @@
+"use client";
+import { StandardPage } from "@/common/StandardPage/StandardPage";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table";
+import { MissingShoeStatus, Shift } from "@/constant/constant";
+import {
+  ColumnDef,
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+import { Plus } from "lucide-react";
+import React, { useMemo } from "react";
+
+const shoes = [
+  {
+    color: "Red",
+    status: MissingShoeStatus.Missing,
+    size: 42,
+    ownerName: "John Doe",
+    ownerPhoneNumber: "+1234567890",
+    ownerAddress: "123 Main St, Springfield, IL",
+    date: "2024-12-09",
+    shift: Shift.ShiftA,
+  },
+  {
+    color: "Blue",
+    status: MissingShoeStatus.Found,
+    size: 38,
+    ownerName: "Alice Smith",
+    ownerPhoneNumber: "+0987654321",
+    ownerAddress: "456 Oak Rd, Rivertown, TX",
+    date: "2024-12-08",
+    shift: Shift.ShiftB,
+  },
+];
+
+type MissingShoes = {
+  color: string;
+  status: string;
+  size: number;
+  ownerName: string;
+  ownerPhoneNumber: string;
+  ownerAddress: string;
+  date: string;
+  shift: Shift;
+};
+
+export const MissingShoes = () => {
+  const columns: ColumnDef<MissingShoes>[] = useMemo(
+    () => [
+      {
+        accessorKey: "ownerName",
+        header: "Owner Name",
+        cell: ({ row }) => (
+          <div className="capitalize">{row.getValue("ownerName")}</div>
+        ),
+      },
+      {
+        accessorKey: "ownerPhoneNumber",
+        header: "Phone Number",
+        cell: ({ row }) => (
+          <div className="capitalize">{row.getValue("ownerPhoneNumber")}</div>
+        ),
+      },
+      {
+        accessorKey: "ownerAddress",
+        header: "Address",
+        cell: ({ row }) => <div>{row.getValue("ownerAddress")}</div>,
+      },
+      {
+        accessorKey: "date",
+        header: "Date",
+        cell: ({ row }) => <div>{row.getValue("date")}</div>,
+      },
+      {
+        accessorKey: "shift",
+        header: "shift",
+        cell: ({ row }) => <div>{row.getValue("shift")}</div>,
+      },
+      {
+        accessorKey: "size",
+        header: "size",
```

Comment:
header size, color & shift not in capital case

Reply:
Fixed